apps/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "private": false,
+  "name": "@defichain-apps/nest-apps",
+  "version": "0.0.0",
+  "description": "A collection of TypeScript + JavaScript tools and libraries for DeFi Blockchain developers to build decentralized finance for Bitcoin",
+  "repository": "DeFiCh/jellyfish",
+  "bugs": "https://github.com/DeFiCh/jellyfish/issues",
+  "license": "MIT",
+  "scripts": {
+    "build": "nest build",
+    "start:legacy-api": "nest start legacy-api",
+    "start:ocean-api": "nest start ocean-api",
+    "start:rich-list-api": "nest start rich-list-api",
+    "clean": "rm -rf dist",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main"
```

Comment:
these scripts would not be applicable since it's mono?

Reply:
Worked for me, I did similar when testing the rich-list-api but do not find it clean, would like to hear better solution.

Reply:
Since we have `start:*` e.g. `start:legacy-api`, we might not need these?

> Personally, when I develop, I never use `start:*`, I always set up tests to test my hypothesis.